src/devices/schneider_electric.ts

```diff
@@ -32,6 +32,37 @@ function indicatorMode(endpoint?: string) {
     });
 }
 
+function fanIndicatorMode() {
+    const description = 'Set Indicator Mode.';
+    return enumLookup({
+        name: 'indicator_mode',
+        lookup: {
+            'always_on': 3,
+            'on_with_timeout_but_as_locator': 4,
+            'on_with_timeout': 5,
+        },
+        cluster: 'clipsalWiserFanSwitchConfigurationCluster',
```

Comment:
@Koenkk i need to make another PR this name was changed. I'll submit one shortly.